traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.HTTP.tpl.html

```diff
@@ -423,6 +425,25 @@ <h3>Current Value</h3>
                             </aside>
                         </div>
                     </div>
+                    <div class="form-group" ng-class="{'has-error': hasError(cacheConfig.slicePluginBlockSize), 'has-feedback': hasError(cacheConfig.slicePluginBlockSize)}">
+                        <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12" for="slicePluginBlockSize">Slice Plugin Block Size<div class="helptooltip">
+                            <div class="helptext">
+                                The block size in bytes that is used for the slice plugin for range requests. Must be between <code>262144</code> - <code>33554432</code>.
+                                Can only be set if the Range Request Handling is set to <code>Use the slice plugin</code>
+                            </div>
+                        </div>
+                        </label>
+                        <div class="col-md-10 col-sm-10 col-xs-12">
+                            <input id="slicePluginBlockSize" name="slicePluginBlockSize" type="number" class="form-control" ng-model="deliveryService.slicePluginBlockSize" min="262144" max="33554432" ng-readonly="dsCurrent.rangeRequestHandling != 3">
+                            <small class="input-error" ng-show="hasPropertyError(cacheConfig.slicePluginBlockSize, 'min')">Too Small (must be greater than or equal to 262144)</small>
+                            <small class="input-error" ng-show="hasPropertyError(cacheConfig.slicePluginBlockSize, 'max')">Too Large (must be smaller than or equal to 33554432)</small>
+                            <small class="input-error" ng-show="dsCurrent.rangeRequestHandling == 3 && dsCurrent.slicePluginBlockSize == null && !(hasPropertyError(cacheConfig.slicePluginBlockSize, 'min') || hasPropertyError(cacheConfig.slicePluginBlockSize, 'max'))">Required</small>
```

Comment:
unfortunately, the way this is done, an empty value when it IS required will not disable the "update" button on the form. i think you can add an ng-required="dsCurrent.rangeRequestHandling == 3" to the input tag and then just go with this:

<small class="input-error" ng-show="hasPropertyError(cacheConfig. slicePluginBlockSize, 'required')">Required</small>